lib/web/apps.go

```diff
@@ -79,35 +80,37 @@ func (h *Handler) clusterAppsGet(w http.ResponseWriter, r *http.Request, p httpr
 		userGroupLookup[userGroup.GetName()] = userGroup
 	}
 
-	var apps types.Apps
+	var appsAndSPs types.AppServersOrSAMLIdPServiceProviders
```

Comment:
there is a test for listing apps in `apiserver_test.go`, i would add some saml apps in there too in the result